bobgit/add/add_target.go

```diff
@@ -0,0 +1,134 @@
+package add
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Benchkram/bob/pkg/filepathutil"
+)
+
+type RepoTargetMap map[string]string
+
+var ErrRepoNotFound = fmt.Errorf("Repository name not found in target path repository list")
+
+type Target struct {
```

Comment:
Comment. What is a target?

Reply:
Target has been renamed to pathspec. Pathspec is as similar as git pathspec just one additional attribute is being handled extra, the repository name. to handle the multiple repositories for bob.